docs/rnw_support.md

```diff
@@ -0,0 +1,618 @@
+# Comprehensive Guide to create simple app using React Native Web and React Native Elements

```

Comment:
this is one of the most detailed and step by step instructions I have seen. Fantastic job on this! 👏 💯